scripts/train/finetune_example/README.md

```diff
@@ -13,14 +13,15 @@ The contents of this directory provide a concrete example of finetuning an LLM o
 Here, we have a minimal example that includes all the necessary pieces:
 - `train.jsonl`: Our local dataset. (It is actually just a snippet of the ARC Easy ICL evaluation set, so it's not something we'd want train on for real.)
 - `preprocessing.py`: A python file that defines the "preprocessing function" we will use to format our dataset into the required "prompt"/"response" structure.
-- `gpt2-arc-easy.yaml`: The configuration YAML for finetuning a pretrained gpt2 model on our local ARC Easy snippet with our custom preprocessing function.
+- `gpt2-arc-easy--cpu.yaml`: The configuration YAML for finetuning a pretrained gpt2 model on our local ARC Easy snippet with our custom preprocessing function. You can run this on CPU.
```

Comment:
can we note how long this will take so ppl don't freak out

Reply:
Done